spec/controllers/recordings_controller_spec.rb

```diff
@@ -134,13 +135,107 @@
     end
   end
 
-  describe '#publish_recording' do
-    it 'Updates Recording with new visibility value' do
+  describe '#update_visibility' do
+    # published (publish: true, protected: false) -> unpublished (publish: false, protected: false)
+    it 'calls the publish request with the right values when changing recording visibility from Published to Unpublished' do
```

Comment:
I would remove the comments and call the test something simpler `it 'moves a recording from Published to Unpublished'`

Reply:
There is 6 tests that checks if the recording is changed on the server (if the requests are being made) and 6 tests that checks if the recording visibility is changed locally. Should I bundle them together? I thought it would be easier by splitting the db from the requests.